Rz/infra migration banner

pacxiu opened the pull request:
# [Infra migration banner](https://app.clubhouse.io/oazo-apps/story/2095/infra-transition-communication-about-transition-to-users)
  
## Changes 👷‍♀️
- adding infra migration banner + "Visit the old Oasis"
- extract `WelcomeAnnouncement` (might be removed or have copy changed after we remove infra banner)
  
![image](https://user-images.githubusercontent.com/19193499/122949211-c2f1d980-d37b-11eb-817d-812a46fea052.png)

  
## How to test 🧪
- run UI, check if banner is visible for all pages
    
## Definition of done ✔️
- [ ] Acceptance criteria for each issue met
- [ ] ~~Unit tests written where needed and passing~~
- [ ] ~~End-to-end tests for happy path~~
- [ ] ~~Documentation updated <When applicable>~~
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

peculiarity commented:
1. Agree with @ferni  regarding the margin under mobile
2. I thought that we should leave the previous message as well
3. The link in oasis-app to the old borrow is purple and in here it's not
4. Not quite sure if this should be visible on the wallet connect page. Please double check with Chris.

![wallet connect](https://user-images.githubusercontent.com/23715025/122953400-f8e48d00-d37e-11eb-98c3-9ce769bd6a82.png)

pacxiu commented:
1. Fixed in both repos
2. For now we should have just 1 banner and have `Visit the old Oasis` inside this one  - so we don't have 2 banners (feedback from Discord). After removing this infra banner old one will be restored, which is hidden now, maybe with copy changed.
3. Unified should be text color in both
4. Added boxShadow here, should be everywhere
